tests/crypto/mbedtls_psa/testcase.yaml

```diff
@@ -0,0 +1,37 @@
+# The goal here is to showcase that Mbed TLS's PSA implementation can be
```

Comment:
I think it could help to make sure that we talk about things naming them properly, i.e. here PSA Crypto instead of just PSA, in order to avoid spreading more confusion to the world.
```suggestion
# The goal here is to showcase that Mbed TLS's PSA Crypto implementation can be
```
And I have the same suggestion for this commit's message. Too bad we can't make suggestions on commit messages.
(Actually, there are many occurences of PSA below that could be changed to PSA Crypto. Won't list them individually.)